index.html

```diff
@@ -45,6 +45,113 @@ <h1>
+          <ul id="lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="see-pro"><bold>See Project</bold></button>
+        </div>
+      </div>
+      <div class="grid-container4">
+        <div class="grid-item-flex">
+          <img src="works-section/project4-img.svg" class="w-image4" alt="image1">
+          <h2 class="m-post2">MULTI-POST STORIES</h2>
+          <h6 class="history4"><span class="canopy">CANOPY</span><span class="spa">&#x2022;Back End Dev</span><span class="date">&#x2022;2015</span></h6> 
+          <p class="paragraph4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul id="lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="see-pro"><bold>See Project</bold></button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="card">
+        <div class="content3">
+          <h1>About<br>Myself</h1>
+          <p p-about>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+          <a href="https://www.linkedin.com/in/lugard-agu-45bb05b6" class="connect">LET'S CONNECT</a>
+          <ul class="media-icons">
+            <li><a href="https://twitter.com/Dsn3kings"><img src="header-section\twitter.svg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/lugard-agu-45bb05b6"><img src="header-section\linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="https://web.facebook.com/lordlugard.agu"><img src="header-section\medium.svg" alt="medium"></a></li>
+            <li><a href="https://github.com/lugard1"><img src="header-section\github.svg" alt="github"></a></li>
+            <li><a href="https://instagram.com/lugardagu"><img src="header-section\peace.svg" alt="peace"></a></li> 
+          </ul> 
+          <button class="resume">Get my resume</button>
+        </div>
+        <div class="languages">
+          <div class="c-lang">
+            <h2>Languages</h2>
+            <img src="contact-images/union-down-arrow.svg" alt="right arrows">
+          </div>
+          <div class="lang-type">
+            <div class="c-js">
+              <a href="#"><img src="languages/JS.svg" class="js-image" alt="JS-logo">JavaScript</a>
+            </div>
+            <div class="c-html">
+              <a href="#"><img src="languages/html.svg" class="html-image" alt="html-logo">HTML</a>
+            </div>
+            <div class="c-css">
+              <a href="#"><img src="languages/css.svg" class="css-image" alt="css-logo">CSS</a>
+            </div>
```

Comment:
- [ ] Kindly make sure your etch stacks are grabbed inside a `ul` and each one of them inside a `li`